### PR TITLE
Refactor of code using variables named "location"

### PR DESCRIPTION
**Pull request by alcen**
This commit is meant as a patch for issue #160. `location` is already defined, used for `Window.location`, and should not be reused.

Variables that were named `location` were changed to either `wkspLocation` or `locat`. One function that used the variable `location` without defining it was refactored such that the correct reference was used.

### CODE REVIEW

**Comment by alcen** [comment]
> Hi can we keep the name consistent everywhere, for example it has a different name here 

Ok, no problem. I have addressed this in the latest commit.
&nbsp;
> In my opinion the issue isn't really isn't an issue. We don't really use window.location

Well, I guess it makes the code more legible :/

**Comment by martin-henz** [comment]
Yes, it's probably better like this. Nothing wrong with explicit names. The name "location" is indeed special in web programming with JavaScript: 
https://www.w3schools.com/jsref/obj_location.asp

So best avoid it.

File: src/components/academy/grading/GradingHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import { Popover, PopoverInteractionKind, Position } from '@blueprintjs/core';
```
Comment (rrtheonlyone) [inline src/components/academy/grading/GradingHistory.tsx]: Why is this file added back?

File: src/utils/history.ts [hunk 2]
```diff
@@ -21,6 +21,6 @@ type lastLocation = string | null;
 
 export const history = createBrowserHistory();
 
-history.listen((location, action) => {
-  store.dispatch(updateHistoryHelpers(location.pathname));
+history.listen((locat, action) => {
```
Comment (rrtheonlyone) [inline src/utils/history.ts]: It is actually correct to use `location` here. Refer to https://github.com/ReactTraining/history#listening

File: src/components/academy/grading/GradingNavLink.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { Classes, Icon } from '@blueprintjs/core';
```
Comment (rrtheonlyone) [inline src/components/academy/grading/GradingNavLink.tsx]: Same for this

**Comment by alcen** [comment]
> I like this change! It indeed makes the naming more explicit and clear 👍
> Some comments:

Thanks for the review! Must have accidentally committed the removed files somewhere in the working tree, oops...

I have made the changes in the latest commit, do take a look!
